rclrs/package.xml

```diff
@@ -13,6 +13,7 @@
   <license>Apache License 2.0</license>
   <author email="esteve@apache.org">Esteve Fernandez</author>
 
+  <build_depend>clang<build_depend>
```

Comment:
`libclang-dev` should be more specific, can you update your PR? Thanks

Reply:
```suggestion
  <build_depend>libclang-dev<build_depend>
```